### PR TITLE
chore: mfa docu fixes

### PR DESCRIPTION
**Pull request by tomekpapiernik**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability.
      If this pull request addresses a security vulnerability,
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

## Further comments

A few quick fixes for the MFA topic - updated formatting and a few headings.

### CODE REVIEW

File: docs/guides/mfa/15_totp.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: totp
-title: Time-Based One-Time Password (TOTP)
+title: TOTP (Google Authenticator)
 ---
 
 You can enable Time-Based One-Time Password authentication in the Ory Identity Service (Kratos) to allow users to perform 2FA with
```
Comment (vinckr) [inline docs/guides/mfa/15_totp.mdx:6]:
```suggestion
You can enable Time-Based One-Time Password (TOTP) authentication in the Ory Identity Service (Kratos) to allow users to perform 2FA with
```

File: docs/guides/mfa/20_webauthn.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: webauthn-fido-yubikey
-title: WebAuthn / FIDO2 (YubiKey)
+title: WebAuthn/FIDO2 (YubiKey)
```
Comment (vinckr) [inline docs/guides/mfa/20_webauthn.mdx:3]: nit
I would prefer no "/" in headings, maybe just 
```suggestion
title: WebAuthn and FIDO2 (YubiKey)
```
?

**Review by vinckr — APPROVED** [review]

File: docs/guides/mfa/15_totp.mdx (hunk 1 above)
Comment (tomekpapiernik) [reply docs/guides/mfa/15_totp.mdx:6]: 👍🏻

File: docs/guides/mfa/20_webauthn.mdx (hunk 2 above)
Comment (tomekpapiernik) [reply docs/guides/mfa/20_webauthn.mdx:3]: Yea, that's okay, we can do it :)

**Review by piotrmsc — APPROVED** [review]

**Pull request merged**